EPA/professional_discussion.md

```diff
@@ -0,0 +1,122 @@
+
+Example questions:
+* What are some of the risks of DevOps? Can you do DevOps wrong?
+* What practices does your team have in place to prevent siloing of knowledge?
+* Who's in charge of DevOps structure in your organisation?
+* What are the risks of CI/CD pipelines?
+
+### Data Ethics
+Pass Criteria:
+* Identifies relevant data protection legislation and assesses its impact on the ethical use of customer data, as well as its relevance to emerging technologies, such as Artificial Intelligence and Machine Learning.
+
+Example Questions:
+* How does your team handle data in an ethical way?
+* What legislation is relevant to data processing within the UK?
+
+### Problem Solving
+Pass Criteria:
+* Identifies different problem solving techniques and evaluates how they use modelling approaches that are best suited to each technique in order to gain consensus as a team.
+
+Distinction Criteria:
+* Describes how they facilitated an incident post- mortem/lessons learned session. 
+* Explains the root cause analysis process. Gains consensus on an improvement plan, including accountabilities and the  implementation timeline.
+
+Example Questions:
+* Describe a problem you've solved recently, and how you approached it
+* How does your organisation learn from incidents in the past?
+* When have you facilitated an incident post-mortem?
+
+### The Profession in Context
+Pass Criteria:
+* Identifies the typical multi-disciplinary team roles and explains how they fit within the organisation and the wider digital landscape.
+* Explains how they completed a task, deploying a flexible, collaborative and pragmatic approach with peers and other stakeholders.
+* Describes examples of different communication methods used when dealing with internal and external stakeholders
+* Explains how they have acted in an inclusive and professional manner.
+
+Example Questions:
+* Tell me about how your team is structured
+* How does your team work?
+* What are some of the key roles with regard to DevOps in your organisation?
+* How do you handle conflicting priorities?
+
+### Tooling & Technology
+Pass Criteria:
+* Explains the difference between the various types of implementation - on premise v SaaS, open source v enterprise, bespoke v off-the-shelf.
+* Explains an example of having utilised the right type of tool for a particular task, describing the pros and cons of the alternatives.
+
+Distinction Criteria:
+* Justifies their choice of tooling and the potential impact of making an alternative choice explaining the cause and effect of making the wrong decision.
+
+Example Questions:
+* How did you choose your CI/CD tool for your project?
+* Tell me about a tool that you use, and what alternatives you've considered
+
+### Continuous Learning & Development
+Pass Criteria:
+* Explains the CPD undertaken by themselves in order to keep up with cutting edge technologies and maintain appropriate certifications.

```

Comment:
Worth expanding the acronym? At least I assume that's what it's supposed to be :D
```suggestion
* Explains the CPD (Continuous Personal Development) undertaken by themselves in order to keep up with cutting edge technologies and maintain appropriate certifications.
```